crates/rattler_conda_types/Cargo.toml

```diff
@@ -23,6 +23,7 @@ purl = { workspace = true, features = ["serde"] }
 rattler_digest = { path = "../rattler_digest", version = "0.19.4", default-features = false, features = ["serde"] }
 rattler_macros = { path = "../rattler_macros", version = "0.19.4", default-features = false }
 regex = { workspace = true }
+simd-json = { version = "*", features = ["serde_impl"]}
```

Comment:
Can you add a minimal version here?

Reply:
made this `workspace=true` and added `simd-json` in root `Cargo.toml`